### PR TITLE
(README) Add services running on HERO

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
 1. Close the current hero-display session
 2. Repeat steps 1-3 & 5 from [First install](#first-install)
 
+
+## HERO1
+### Services
+The following services are running on HERO1. Normally these are runned from the demo account:
```
Comment (LarsJanssenTUe) [inline README.md]:
```suggestion
The following services are running on HERO1. Normally these are ran from the demo account:
```

File: README.md [hunk 2]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
...
+- **hero1-monitor.service**: HERO version of `hsrb_monitor` from `hsrb_bringup`
+- **hero1-ntpdate.service**: Service to sync time with servers
+- **hero1-phone-ui.service**: Service to always have the webserver running
+- **hero1-roscore.service**: Service containing the roscore
+- **hero1-speech1-vboxautostart-service**: Slightly changed version of vboxautostart
+- **hero1-speech2-vboxballoonctrl-service**: Slightly changed version of vboxballoonctrl
+- **hero1-speech3-vboxdrv**: Slightly changed version of vboxdrv
+- **hero1-speech4-vboxweb-service**: Slightly changed version of vboxweb
+- **hero1-speech5-windows-speech**: Service to start the Windows virtualbox
+
```
Comment (LarsJanssenTUe) [inline README.md]: If I understand correctly these are all related to the virtualbox and should therefore be obsolete if we get rid of the VB?

File: README.md [hunk 3]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
...
 ## HERO2
+### Services
+The following services are running on HERO1. Normally these are runned from the demo account:
```
Comment (LarsJanssenTUe) [inline README.md]:
```suggestion
The following services are running on HERO1. Normally these are ran from the demo account:
```

File: README.md [hunk 4]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
...
+- **hero2-battery-manager**: Running a battery manager to let the robot speak in case of a low battery level
+- **hero2-openpose**: Running OpenPose
```
Comment (LarsJanssenTUe) [inline README.md]: Why does this service always run?

File: README.md [hunk 5]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
 1. Close the current hero-display session
 2. Repeat steps 1-3 & 5 from [First install](#first-install)
 
+
+## HERO1
+### Services
+The following services are running on HERO1. Normally these are runned from the demo account:
+- **hero1-battery-conversion**: Publishing a BatteryState message from the custom Toyota messages
+- **hero1-monitor.service**: HERO version of `hsrb_monitor` from `hsrb_bringup`
```
Comment (PetervDooren) [inline README.md]: briefly explain what the monitor does

**Comment by LarsJanssenTUe** [comment]
Oah and please look at why CI fails!

File: README.md [hunk 6]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
...
+- **hero1-ntpdate.service**: Service to sync time with servers
+- **hero1-phone-ui.service**: Service to always have the webserver running
```
Comment (PetervDooren) [inline README.md]:
```suggestion
- **hero1-phone-ui.service**: Service to always have the webserver running to facilitate the hero-dashboard
```

File: README.md [hunk 7]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
...
+- **hero1-roscore.service**: Service containing the roscore
```
Comment (PetervDooren) [inline README.md]: technically this launches more than just the roscore but I would need to check the toyota code to see what it does exactly

File: README.md (hunk 3 above)
Comment (PetervDooren) [inline README.md]:
```suggestion
The following services are running on HERO2. Normally these are runned from the demo account:
```

File: README.md (hunk 4 above)
Comment (PetervDooren) [reply README.md]: It shouldn't in my opinion.

File: README.md [hunk 8]
```diff
@@ -77,7 +77,29 @@ The following steps can be done via SSH or directly on the robot (SSH: `ssh admi
...
+- **hero1-speech3-vboxdrv**: Slightly changed version of vboxdrv
+- **hero1-speech4-vboxweb-service**: Slightly changed version of vboxweb
+- **hero1-speech5-windows-speech**: Service to start the Windows virtualbox
+
 ## HERO2
+### Services
+The following services are running on HERO1. Normally these are runned from the demo account:
+- **hero2-battery-manager**: Running a battery manager to let the robot speak in case of a low battery level
+- **hero2-openpose**: Running OpenPose
+- **hero2-rgbd-shm-server**: Running a `rgbd` shared memory server, so there is only one RGBD connection from HERO1 to HERO2
```
Comment (PetervDooren) [inline README.md]: Why do we need to launch this from a service again? It is not needed for battery management.

File: README.md (hunk 2 above)
Comment (MatthijsBurgh) [reply README.md]: Yes

File: README.md (hunk 4 above)
Comment (MatthijsBurgh) [reply README.md]: That is not the scope of this PR. This PR is documentation.

File: README.md (hunk 5 above)
Comment (MatthijsBurgh) [reply README.md]: Nope, Find out yourself in the launch file.

File: README.md (hunk 4 above)
Comment (PetervDooren) [reply README.md]: https://github.com/tue-robotics/hero_bringup/issues/41

File: README.md (hunk 8 above)
Comment (MatthijsBurgh) [reply README.md]: No, but handy that is is running for all users. This is on the same level as the camera. You want this always running.

File: README.md (hunk 7 above)
Comment (MatthijsBurgh) [reply README.md]: Added more info

File: README.md (hunk 3 above)
Comment (MatthijsBurgh) [reply README.md]: Fixed

**Comment by LarsJanssenTUe** [comment]
LGTM, I leave it to @PetervDooren to have a final say. But I think it is fine!

**Pull request merged**